Apps/W1/HybridGP/app/src/pages/GPMigrationConfiguration.Page.al

```diff
@@ -623,6 +638,9 @@ page 4050 "GP Migration Configuration"
                 Caption = 'Upgrade settings';
                 ToolTip = 'Change the settings for the GP upgrade.';
                 RunObject = page "GP Upgrade Settings";
+                Promoted = true;
+                PromotedCategory = Process;
```

Comment:
We need to use new syntax for promoted actions. There is a page action that fixes this.

Reply:
Thanks, will make this change.